pkg/virtctl/vnc/vnc.go

```diff
@@ -22,41 +22,52 @@ package vnc
 import (
 	"fmt"
 	"io"
-	"log"
 	"net"
 	"os"
 	"os/exec"
 	"os/signal"
 
 	flag "github.com/spf13/pflag"
-	kubev1 "k8s.io/api/core/v1"
+
+	"github.com/spf13/cobra"
+
+	"k8s.io/client-go/tools/clientcmd"
+
+	"github.com/golang/glog"
```

Comment:
should we be trying to use our logging package somehow here instead of glog?

Reply:
i think the real question here is should our log pkg be a wrapper for glog?

Reply:
My plan always was to use glog output format clientside (virtctl) and structured logging server side (virt-controller, virt-handler, virt-launcher). I would replace the glog package completely in our vendor folder and add a switch, so that we can select the actual logging format on application startup.